### PR TITLE
Rc 2.0.1

### PR DESCRIPTION
**Pull request by Salem-Luos**

### CODE REVIEW

**Review by BiliouriV — APPROVED** [review]

**Review by ncros3 — APPROVED** [review]
That's ok for me.

File: Apps/Gate/gate.c [hunk 1]
```diff
@@ -95,8 +96,7 @@ void Gate_Loop(void)
                 {
                     // This is the first time we perform a convertion
                     // Evaluate the time needed to convert all the data of this configuration and update refresh rate
-                    uint32_t execution_time = ((Luos_GetSystick() - last_time) * 2) + 2;
-                    update_time             = TimeOD_TimeFrom_ms(execution_time);
+                    update_time = (float)RoutingTB_BigestID() * 0.01;
```
Comment (nicolas-rabault) [inline Apps/Gate/gate.c]: 10 ms for each services seems really eavy. what about 1ms?

**Review by nicolas-rabault — CHANGES_REQUESTED** [review]
Please take some time to make the commit comment as clear as possible.
In example we start commit comment by [example] allowing to know witch example is modified.
I think 10ms per service on Gate is too heavy.

File: Apps/Gate/gate.c (hunk 1 above)
Comment (ncros3) [reply Apps/Gate/gate.c]: I changed this according to your proposal.

Comment (JeromeGalan) [reply Apps/Gate/gate.c]: Force merge as N. Rabault request changes has been implemented

**Pull request merged**